subsystem/kubeapi_test.go

```diff
@@ -55,8 +55,9 @@ const (
 
 var (
 	imageSetsData = map[string]string{
-		"openshift-v4.7.0": "quay.io/openshift-release-dev/ocp-release:4.7.2-x86_64",
-		"openshift-v4.8.0": "quay.io/openshift-release-dev/ocp-release:4.8.0-fc.0-x86_64",
+		"openshift-v4.7.0":     "quay.io/openshift-release-dev/ocp-release:4.7.2-x86_64",
+		"openshift-v4.8.0":     "quay.io/openshift-release-dev/ocp-release:4.8.0-fc.0-x86_64",
```

Comment:
nit: Lets update to latest z release if possible: (4.8.26, 4.7.39). The tab looks like a typo. Will you need to add x86 version for 4.9?

Reply:
There's not really a point to chase the x releases here because all we test is a basic flow. There are different e2e test for the latest release in the CI